DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml

```diff
@@ -28,31 +28,52 @@
 
   @foreach (var competency in Model)
   {
-<tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
-  <tr role="row" class="nhsuk-table__row first-row">
-    <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
-      <span class="nhsuk-table-responsive__heading">@competency.Vocabulary </span>@competency.Name
-      @if (competency.AlwaysShowDescription) {
-    <p class="nhsuk-u-margin-top-2">@Html.Raw(competency.Description)</p>
-      }
-    </td>
-    <partial name="Shared/_AssessmentQuestionOverviewCells" model="competency.AssessmentQuestions.First()" view-data="@(new ViewDataDictionary(ViewData) { { "competencyNumber", competency.RowNo } })" />
-    <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell nhsuk-u-font-size-16">
-      <span class="nhsuk-table-responsive__heading">Action </span>
-      <a asp-action="SelfAssessmentCompetency" asp-route-selfAssessmentId="@ViewContext.RouteData.Values["selfAssessmentId"]" asp-route-competencyNumber="@competency.RowNo">
-        Review
-      </a>
-    </td>
-  </tr>
-  @foreach (var question in competency.AssessmentQuestions.Skip(1))
-  {
-    <tr role="row" class="nhsuk-table__row">
-      <partial name="Shared/_AssessmentQuestionOverviewCells" model="question" view-data="@(new ViewDataDictionary(ViewData) { { "competencyNumber", competency.RowNo } })" />
+    <tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
+      <tr role="row" class="nhsuk-table__row first-row">
+        <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
+          @if (competency.Description != null && !competency.AlwaysShowDescription)
+          {
+            <details class="nhsuk-details">
+              <summary class="nhsuk-details__summary">
+                <p class="nhsuk-heading-l nhsuk-u-font-size-24 nhsuk-u-margin-bottom-0">
+                  <span class="nhsuk-table-responsive__heading">@competency.Vocabulary </span>
+                  <span class="nhsuk-details__summary-text">@competency.Name</span>
+                </p>
+              </summary>
+              <div class="nhsuk-body-l nhsuk-details__text nhsuk-u-margin-left-6 nhsuk-u-margin-top-2">
+                @(Html.Raw(@competency.Description))
+              </div>
+            </details>
+          }
+          else
+          {
+            <p class="nhsuk-heading-l nhsuk-u-margin-bottom-0 nhsuk-u-font-size-24">
```

Comment:
Can we get rid of the `nhsuk-heading-l nhsuk-u-font-size-24` classes from here, too please.